src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -168,7 +177,7 @@ def make_results(
         func = build_results
         kwargs = {}
     main = handle_exceptions(func=func, logger=logger, with_debugger=with_debugger)
-    main(raw_output_dir, final_output_dir, mark_best, test_run, artifact_path, **kwargs)
+    main(raw_output_dir, final_output_dir, mark_best, test_run, seed, artifact_path, **kwargs)
```

Comment:
I think this is going to break the `-j` stuff. I'll fix later if needed

Reply:
Why would this break `-j`? There's a default value, so seed will just be the empty string in that case.

Reply:
Good call. Should be fixed now, though we aren't using the argument in the jobmon workflow at all yet.